infrastructure/templates/cronjob.yaml

```diff
@@ -1,49 +1,53 @@
-  namespace: loadtest
-spec:
-  schedule: {{ $job_options.schedule }}
-  suspend: true
-  jobTemplate:
-    spec:
-      completions: {{ $job_options.jobsParallelism }}
-      parallelism: {{ $job_options.jobsParallelism }}
-      template:
-        metadata:
-          labels:
-            test: loadtest-{{ $job_options.serviceName }}
-            pod: {{ $job_name }}
-        spec:
-          containers:
-          - name: {{ $job_name }}
-            image: {{ $job_options.image }}
-            imagePullPolicy: Always
-            env:
-              - name: BBB_SECRET
-                valueFrom:
-                  secretKeyRef:
-                    name: bbb-secret
-                    key: bbb-secret
-              - name: BIGBLUEBOT_HOST
-                value: {{ $job_options.host }}
-              - name: BIGBLUEBOT_FETCH_HOST
-                value: {{ $job_options.fetchHost }}
-              - name: BIGBLUEBOT_VERSION
-                value: {{ $job_options.bbbVersion }}
-              - name: BIGBLUEBOT_ROOM
-                valueFrom: 
-                  secretKeyRef:
-                    name: bbb-secret
-                    key: bbb-room
-              - name: BIGBLUEBOT_BOTS
-                value: {{ $job_options.bbbBots }}
-              - name: BIGBLUEBOT_WAIT
-                value: {{ $job_options.botLifeDuration }}
-              - name: BIGBLUEBOT_LIFE
-                value: {{ $job_options.botJoinGaps }}
-          restartPolicy: OnFailure
----
+{{- range $job_name, $job_options := .Values.cronjobs }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: loadtest-bbb-{{ $job_name }}
+  namespace: loadtest
+spec:
+  schedule: {{ $job_options.schedule | quote }}
+  suspend: true
+  jobTemplate:
+    spec:
+      completions: {{ $job_options.jobsParallelism }}
+      parallelism: {{ $job_options.jobsParallelism }}
+      template:
+        metadata:
+          labels:
+            test: loadtest-{{ $job_options.serviceName }}
+            pod: {{ $job_name }}
+        spec:
+          containers:
+          - name: {{ $job_name }}
+            image: {{ $job_options.image }}
+            imagePullPolicy: Always
+            args:
+              ['./loadtest/tests/{{ $job_options.scriptPath }}']
+            env:
+              - name: BRANCH
+                value: DBP-193-Extend-BBB-loadtest-with-Audio-and-Video
+              - name: BIGBLUEBOT_SECRET
+                valueFrom:
+                  secretKeyRef:
+                    name: bbb-secret
+                    key: userSecret
+              - name: BIGBLUEBOT_HOST
+                value: {{ $job_options.host }}
+              - name: BIGBLUEBOT_FETCH_HOST
+                value: {{ $job_options.fetchHost }}
+              - name: BIGBLUEBOT_VERSION
+                value: {{ $job_options.bbbVersion | quote }}
+              - name: BIGBLUEBOT_ROOM
+                valueFrom:
+                  secretKeyRef:
+                    name: bbb-secret

```

Comment:
Bitte den secret.yaml file mit pushen, die secrets an sich sollen in Datein lokal bei dir bleiben(Nicht in dem secret.yaml an sich) damit andere Nutzer wissen welche Dateien sie bei dich anlegen müssen

Reply:
secret.yaml ist gepushed aber ohne keys. Habe es wieder in gitignore restored so das es nicht aus versehen gepushed wird.